x/qgb/abci_test.go

```diff
@@ -64,7 +64,7 @@ func TestValsetCreationWhenValidatorUnbonds(t *testing.T) {
 	assert.Equal(t, currentAttestationNonce+1, pk.GetLatestAttestationNonce(ctx))
 }
 
-func TestValsetChangeWhenEditingOrchAddr(t *testing.T) {
+func TestEditingOrchAddr(t *testing.T) {
```

Comment:
sorry for my previously imprecise feedback. I think it may be preferable if the test name described the function under test. In https://go.dev/doc/tutorial/add-a-test and https://www.digitalocean.com/community/tutorials/how-to-write-unit-tests-in-go-using-go-test-and-the-testing-package and in most cases in this repo, a test name matches the function that is being tested. I don't think this is a strict requirement but would be beneficial here because this test contains so many lines that it wasn't clear to me as an unfamiliar reader which line of code is the behavior this test is testing. On re-review, it seems like this test is testing the behavior of qgb.EndBlocker, specifically https://github.com/celestiaorg/celestia-app/blob/8604be960b0565df66a6c4da79371f713ca62798/x/qgb/abci.go#L35

If this is the case, what are your thoughts on using a test name like

```suggestion
func TestEndBlockerAfterEditingOrchestratorAddress(t *testing.T) {
```

I think a more ideal approach would be:
```go
func TestEndBlocker(t *testing.T) {
	testCases := []testCase{{
		name:       "after editing orchestrator address", 
		// ...  
		},
	}
}
```
but we may not be able to do that given your other comment on difficulty refactoring to table tests

Reply:
Thanks a lot for the feedback.

The table tests using the same in-process network result in flakiness. That's why I didn't add them.